src/main/java/mods/railcraft/client/particle/ParticleBaseSmokeShrinking.java

```diff
@@ -13,6 +13,7 @@ protected ParticleBaseSmokeShrinking(ClientWorld world, double x, double y, doub
   @Override
   public float getQuadSize(float partialTicks) {
     return this.quadSize * MathHelper.clamp(
-        ((float) this.age + partialTicks) / (float) this.lifetime * 32.0F, 0.0F, 1.0F);
```

Comment:
This cast might be necessary to make sure we get a float result and not an integer.

Reply:
removing soon:tm: